README.md

```diff
@@ -60,8 +60,14 @@ Before start, you need to meet the following requirements:
 - See [hardware resource requirements](https://github.com/Microsoft/pai/wiki/Resource-Requirement).
 
 #### 2 Deploy OpenPAI
-##### 2.1 [Customized deploy](./docs/pai-management/doc/cluster-bootup.md#customizeddeploy)
-##### 2.2 [Single Box deploy](./docs/pai-management/doc/cluster-bootup.md#singlebox)
+
+If you only have one mahince, and want to deploy pai on it. Please choose ```Single Box deploy``` following. 
+
+If you have a cluster which contains more than 2 machine and want to deploy pai on it. Please choose ```Distributed deploy``` following.
+
+##### 2.1 [Single Box deploy](./docs/pai-management/doc/single-box.md)
```

Comment:
Distributed deploy 's priority and use case is > single box deploy, suggest turn the order make Distributed deploy as 2.1

Reply:
Put the simple one at first will be better.

Reply:
@squirrelsc @scarlett2018 
what's your suggestion?

I think it is better to sort the order by deploymnet and using frequency (distributed > single box).